opsdroid/core.py

```diff
@@ -318,24 +348,42 @@ def _connector_names(self):  # noqa: D401
         return names
 
     async def start_databases(self, databases):
-        """Start the databases."""
+        """Start the databases.
+
+        Iterates through all the database modules parsed 
+        in the argument, connects and starts them.
+
+        Args:
+            databases (list): A list of all database modules to be started.
+
+        """
         if not databases:
             _LOGGER.debug(databases)
             _LOGGER.warning(_("All databases failed to load"))
         for database_module in databases:
             for name, cls in database_module["module"].__dict__.items():
                 if (
-                    isinstance(cls, type)
-                    and issubclass(cls, Database)
-                    and cls is not Database
+                    isinstance(cls, type) and
+                    issubclass(cls, Database) and
+                    cls is not Database
                 ):
                     _LOGGER.debug(_("Adding database: %s"), name)
                     database = cls(database_module["config"])
                     self.memory.databases.append(database)
                     await database.connect()
 
     async def run_skill(self, skill, config, message):
-        """Execute a skill."""
+        """Execute a skill.
+
+        Attempts to run the skill parsed and provides other arguments to the skill if necessary.
+        Also handles the exception encountered if th e
+
+        Args:
+            skill: name of the skill to be run.
+            config: The configuration the skill must be loaded in.
+            message: Message to be parsed to the chat service.
```

Comment:
I'm not so sure about the role of the `message` argument. Some guidance on this would be most appreciated!